src/TestSuite/IntegrationTestCase.php

```diff
@@ -824,7 +824,7 @@ public function assertResponseEquals($content, $message = '')
         if (!$this->_response) {
             $this->fail('No response set, cannot assert content. ' . $message);
         }
-        $this->assertEquals($content, $this->_response->body(), $message);
+        $this->assertEquals($content, (string)$this->_response->getBody(), $message);
```

Comment:
@markstory ``(string)$this->_response->getBody()`` seems to be re-ocurring thing. We could extract it to a ``IntegrationTestCase::getBodyAsString()`` method, to ease userland tests even further.